### DISCUSSION TITLE
Adding different torrent client(s)

### DISCUSSION THREAD

**Discussion by Ivar418** [haugene/docker-transmission-openvpn#2818] [Ideas]
I have been having issues with using transmission because of https://github.com/transmission/transmission/issues/2462 and i moved over to qbittorrent in another container. Prefferably i would set the port forward there an not transmission because it is not used naymore.

Since qbittorrent ads some features that i like like a default tracker list thats added to every torrent, maybe it is a nice idea to add support for differen torrent clietns, of maybe the ability to edit the script that sets the listen port in transmission so that it sets it in a different client.

I already made a lil script that gets the port forward and the sets it in qbittorrent via its api, but since transmission will already have owned that port it cannot also use it. I tried too to set any of the custom scripts to set the port before transmission can "steal" it but since qbittorrent depends on this docker it is not up yet during that time

**Comment by ilike2burnthing** [comment]
https://hub.docker.com/r/binhex/arch-qbittorrentvpn
or
https://hub.docker.com/r/trigus42/qbittorrentvpn
are going to be your best options.
